settings/DevHome.Settings/Models/Setting.cs

```diff
@@ -13,7 +14,7 @@ public class Setting
 
     public string Path { get; }
 
-    public string FullName { get; }
+    public string UniqueId { get; }

```

Comment:
I might have missed this, but, how does Settings enforce uniquiness?

Reply:
Via the uniqueId.  It comes from the appextension stuff.